bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Sequence.cpp

```diff
@@ -267,44 +267,52 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Trajectory_Sequence(pybind11::module
             .def(
                 "add_coast_segment",
                 &Sequence::addCoastSegment,
-                arg("event_condition"),
                 R"doc(
                     Add a coast segment.
 
                     Args:
                         event_condition (EventCondition): The event condition.
 
-                )doc"
+                )doc",
+                arg("event_condition")
             )
             .def(
                 "add_maneuver_segment",
                 &Sequence::addManeuverSegment,
-                arg("event_condition"),
-                arg("thruster_dynamics"),
                 R"doc(
                     Add a maneuver segment.
 
                     Args:
                         event_condition (EventCondition): The event condition.
                         thruster_dynamics (Thruster): The thruster dynamics.
 
-                )doc"
+                )doc",
+                arg("event_condition"),
+                arg("thruster_dynamics")
             )
 
             .def(
                 "solve",
                 &Sequence::solve,
-                arg("state"),
                 R"doc(
                     Solve the sequence.
 
+                    This method will terminate early if the maximum propagation duration is exceeded.
+                    If an event condition is provided, it will terminate if the event condition is met.
```

Comment:
```suggestion
                    If an event condition is provided, it will terminate when the event condition is met.
```

mega super ultra nit

Reply:
So technically the condition could never be met. Therefore I put an `if`.